images/dcos/bootstrap.sh

```diff
@@ -84,23 +79,19 @@ echo "* service proxy: $service_proxy"
 etcd_server_port=${ETCD_SERVER_PORT:-4001}
 etcd_server_peer_port=${ETCD_SERVER_PEER_PORT:-4002}
 
-ENABLE_ETCD_SERVER=${ENABLE_ETCD_SERVER:-false}
-if [ "${ENABLE_ETCD_SERVER}" != true ]; then
-  etcd_server_list=${ETCD_SERVER_LIST:-http://${service_proxy}:${etcd_server_port}}
-else
-  etcd_advertise_server_host=${ETCD_ADVERTISE_SERVER_HOST:-127.0.0.1}
-  etcd_server_host=${ETCD_SERVER_HOST:-127.0.0.1}
+ETCD_MESOS=${ETCD_MESOS:-disabled}
```

Comment:
changes in this boostrap file are etcd-mesos specific. the name ETCD_MESOS aligns closely with the dcos config option name that we're adding in multiverse. the goal is to keep the multiverse config options as simple as possible, not to make this bootstrap image as flexible as possible. `disabled` will run an internal etcd server in the uber container. any other value is the name of the etcd-mesos framework that we should attach to.